checkov/main.py

```diff
@@ -28,10 +29,10 @@
 logging_init()
 
 
-def run(banner=checkov_banner):
+def run(banner=checkov_banner, argv=sys.argv[1:]):
     parser = argparse.ArgumentParser(description='Infrastructure as code static analysis')
     add_parser_args(parser)
-    args = parser.parse_args()
+    args = parser.parse_args(argv or ['--help'])
```

Comment:
Not sure if we want the `or ['--help']` I normally find it useful, but up to you

cc @schosterbarak

Reply:
Hi, Haven't worked that way in the past. 
@KevinHock Can you elaborate on what it means?

Reply:
I'll change this back to what it was

It turns out calling `checkov` with no args gives the following, which we wouldn't want to override with `--help`:

```
% checkov       

       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 1.0.695 
Visualize and collaborate on security issues with Bridgecrew! 
Bridgecrew's dashboard allows automation of future checks, Pull Request scanning and auto-comments, automatic remidiation PR's and more! 
 Plus it's free for 100 cloud resources and a great way to visualize and collaborate on Checkov results. For more information on dashboard integration, see: http://bridge.dev/checkov-dashboard 
 
 To instantly see future Checkov scans in the platform, Press y! 

Visualize results? (y/n): y

Email Address? 
```

Reply:
I `pip install -e .`d the version I have now and got the following, which we could override with `--help` automatically if we wanted to:

```
% checkov


       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 1.0.693 
Update available 1.0.693 → 1.0.695
Run pip3 install -U checkov to update 

No argument given. Try ` --help` for further information
```

Reply:
@KevinHock we should keep the current behaviour of when no args are given (prompt to integrate into bridgecrew platform).

Reply:
Yup, I've removed, sgtm, thanks!